bot/interactions/commands/inventory/inventory.ts

```diff
@@ -36,13 +49,94 @@ export const inventory = {
       });
     }
 
-    // const wallet = await createWallet(interaction.user.id, guildId);
-    // const inventory = await prisma.shopItem.findMany({
-    //     where: {
-    //         walletId: wallet.id
-    //     }
-    // })
+    const interactionReplyOption = await createEmbed(
+      interaction.user,
+      interaction.guild,
+    );
 
-    // const tools = inventory.filter((tool) => tool.type === ItemType.Tools);
+    return await interaction.reply(interactionReplyOption);
   },
 } satisfies Command;
+
+export const inventoryDisposeMenuContext = z.object({
+  walletId: z.string(),
+  type: z.nativeEnum(ItemType),
+});
+
+export const createEmbed = async (
+  user: User,
+  guild: Guild,
+  content?: string,
+) => {
+  const guildId = guild.id;
+  const wallet = await createWallet(user.id, guildId);
+
+  const inventory = await prisma.shopItem.findMany({
+    where: {
+      walletId: wallet.id,
+    },
+  });
+
+  const fields: APIEmbedField[] = [];
+
+  const embed = new EmbedBuilder()
+    .setTitle("Player Inventory")
+    .setDescription("### Tools")
+    .setColor(Colors.Info);
+
+  const tools = inventory.filter((tool) => tool.type === ItemType.Tools);
+  if (tools.length < 1) {
+    embed.addFields([{ name: "No tools data found!", value: "\u200b" }]);
+    return { content: "", embeds: [embed], components: [] };
+  }
+
+  const inventoryDisposeInteraction = await prisma.interaction.create({
+    data: {
+      type: InteractionType.InventoryDisposeToolMenu,
+      guildId,
+      userDiscordId: user.id,
+      payload: JSON.stringify({
+        walletId: wallet.id,
+        type: ItemType.Tools,
+      } satisfies z.infer<typeof inventoryDisposeMenuContext>),
+    },
+  });
+
+  const firstRow = new ActionRowBuilder<StringSelectMenuBuilder>();
+
+  const stringSelectMenuBuilder = new StringSelectMenuBuilder()
+    .setCustomId(inventoryDisposeInteraction.id)
+    .setPlaceholder("Select the item you would like to dispose.");
+
+  /**Filtering out the first 25 items will be added in the future when more tools/items are added */
```

Comment:
JSDoc without ident?
```suggestion
  // Filtering out the first 25 items will be added in the future when more tools/items are added
```